scripts/generate_test_tsv.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/python
+
+import click
+import shutil
+import pandas as pd
+import mmeds.config as fig
+from mmeds.util import join_metadata, write_metadata, load_metadata
+from random import randrange, getrandbits
+from pathlib import Path
+import numpy as np
+import datetime as dt
+
+__author__ = "Adam Cantor"
+__copyright__ = "Copyright 2021, The Clemente Lab"
+__credits__ = ["Adam Cantor", "Jose Clemente"]
+__license__ = "GPL"
+
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('-p', '--path', required=True, type=click.Path(exists=True),
+              help='Path to put the created test files')
+def main(path):
+    file_path = Path(path) / 'validation_files'
+    print('Output: ' + str(file_path))
+    for test_file in file_path.glob('validate_*'):
+        print('Deleting {}'.format(test_file))
+        Path(test_file).unlink()
+
+    # Create the subjects' test files
+    sub_df = pd.read_csv(Path(path) / Path(fig.TEST_SUBJECT_SHORT).name, sep='\t', header=[0,1], na_filter=False)
```

Comment:
```suggestion
    sub_df = pd.read_csv(fig.TEST_SUBJECT_SHORT, sep='\t', header=[0,1], na_filter=False)
```
`fig.TEST_SUBJECT_SHORT` contains the full path to that file when mmeds is properly installed. Here you're getting the file name from it but looking for the file in the location you'll be writing the new files which could be some where other that `test_files`